### PR TITLE
update "Fix #253: Use AdcStart() before each ReadAnalog value"

### LINKED ISSUE #253: ReadAnalog fails
I am using MCCI 4610 and see "?CatenaStm32L0::ReadAnalog(3): CatenaStm32L0_ReadAnalog() failed (5)" messages as in https://github.com/mcci-catena/Catena-Arduino-Platform/issues/125

This happens not all the time and seems to start happening only after some time.

Board Version: 2.7.0
System Clock: 24 MHz

Libraries:
| URL | Commit | Commit Date |
| --- | -----  | ----------- |
| https://github.com/mcci-catena/arduino-lmic.git  | 6fe04ec | Tue, 12 May 2020 09:16:47 -0400  |
| https://github.com/mcci-catena/arduino-lorawan.git  | 4bc0d48 | Sat, 9 May 2020 12:38:28 -0400  |
| https://github.com/mcci-catena/Catena-Arduino-Platform.git  | 92019ca | Tue, 12 May 2020 01:34:08 -0400  |
| https://github.com/mcci-catena/Catena-mcciadk.git  | a428006 | Sat, 21 Dec 2019 20:45:26 -0500  |
| https://github.com/mcci-catena/MCCI_FRAM_I2C.git  | f0a5ea5 | Sat, 21 Dec 2019 16:17:01 -0500  |

I read the battery voltage on every iteration (after waking up from deep sleep):
```
  // vBat
  int vbat_mv = (int)(gCatena.ReadVbat() * 1000.0f);
  gCatena.SafePrintf("vBat:    %d mV\n", vbat_mv);
```
Attached is the log with the relevant lines.
[ReadAnalog-errors.txt](https://github.com/mcci-catena/Catena-Arduino-Platform/files/4763260/ReadAnalog-errors.txt)

### PR DESCRIPTION
**Pull request by dhineshkumarmcci**
Advanced the version number in `README.md` and `CatenaBase.h`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1380,7 +1380,8 @@ This sketch demonstrates the use of the Catena FSM class to implement the `Turns
 
 - v0.19.0 includes the following changes.
 
-  - [#248](https://github.com/mcci-catena/Catena-Arduino-Platform/issues/248) Add interactive command `lorawan configure` to display all hte parameters.
+  - [#253](https://github.com/mcci-catena/Catena-Arduino-Platform/issues/253) Add `AdcStart()` before each `AdcGetValue()` to read Channel value (0.19.0.10).
```
Comment (terrillmoore) [inline README.md:1383]: This must be as follows:

```markdown
- HEAD includes the following changes.

   - Fix [#253](https://github.com/mcci-catena/Catena-Arduino-Platform/issues/253): add `AdcStart()` before each `AdcGetValue()` to read channel value (version 0.19.0.10).

- v0.19.0 includes the following changes.

   - [#248](https://github.com/mcci-catena/Catena-Arduino-Platform/issues/248) Add interactive command `lorawan configure` to display all the parameters.
```

Reason: if you checkout the tag `v0.19.0` you *won't* get the fix for #253.

Comment (dhineshkumarmcci) [reply README.md]: Yes I understand that tag `v0.19.0` does not have the fix for #253 . Sorry for the inconvenience, I have updated the Revision History. I understand how it works and helps for the user to follow. Thanks Terry!

**Comment by dhineshkumarmcci** [comment]
@terrillmoore changes in the file `CatenaStm32L0_ReadAnalog.cpp` of the branch `issue253` is not reflected in the `master` during the merge of this pull-request. 
https://github.com/mcci-catena/Catena-Arduino-Platform/blob/88a771a198371a233de3b6867df86c9fd5d03ba1/src/lib/stm32/stm32l0/CatenaStm32L0_ReadAnalog.cpp#L174-L177
Trying to create a new pull-request with branch `issue253` says no changes to compare. Is that ok to create a new branch and submit a pull-request?

**Comment by terrillmoore** [comment]
It would be best to understand how this happened. I suggest that you fork this repo, first apply your pull request there, then you can push changes upstream from your repo once you are sure they are correct.

**Comment by terrillmoore** [comment]
I think the merge didn't pick up your changes -- you committed changes to issue253 *after* I merged. Reopen #253, rename your local branch to "issue253a" or something, and submit a new pr.

**Comment by terrillmoore** [comment]
Note this from the history above:
image
The highlighted commits appear *after* the [Merged] indication.  As far as I know, there is no way to merge the extra commits without a new PR. They don't point to the reference lines. 

More research points to #254. I *reverted that*, because the documentation was not done.

Proper process was not followed. Proper process is to push additional changes to the branch used for PR #254, not to create a new branch. I am going to REVERT this PR as well, and you must push ALL your changes onto either #254, or a different new PR. Please get review from @mcciyssaroha for this, thanks.

**Pull request merged**